### PR TITLE
[000] Initial Typescript Repo

### PR DESCRIPTION
**Pull request by TaylorJCrawford**

### CODE REVIEW

File: app.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    }
+}
+
+
+app.listen(3000, () => {
+    console.log('Server listening on port 3000');
+});
+
+    // Express Routes
+app.get('/home', (req: Request, res: Response) => {
```
Comment (glenn-hamilton) [inline app.ts]: This should be `/`

File: package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    "url": "git+https://github.com/TaylorJCrawford/ui-localghosts.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/TaylorJCrawford/ui-localghosts/issues"
+  },
+  "homepage": "https://github.com/TaylorJCrawford/ui-localghosts#readme",
+  "dependencies": {
+    "@types/express-session": "^1.17.9",
```
Comment (glenn-hamilton) [inline package.json]: This should be a dev dependency

File: views/templates/master.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Template Home</title>
```
Comment (glenn-hamilton) [inline views/templates/master.html]: Should this title be dynamic as it is a template?

File: app.ts (hunk 1 above)
Comment (miagreen23) [reply app.ts]: Amended

File: package.json (hunk 2 above)
Comment (miagreen23) [reply package.json]: Amended

File: views/templates/master.html (hunk 3 above)
Comment (miagreen23) [reply views/templates/master.html]: Amended

**Review by glenn-hamilton — APPROVED** [review]

**Pull request merged**